nvflare/apis/impl/controller.py

```diff
@@ -66,7 +66,7 @@ def _check_inputs(task: Task, fl_ctx: FLContext, targets: Union[List[Client], Li
 
 
 class Controller(Responder, ControllerSpec, ABC):
-    def __init__(self, task_check_period=0.5):
+    def __init__(self, task_check_period=0.01):
```

Comment:
will this change impact other pipelines / shall we change it for all? - may need comments from Chester / Yan

Reply:
This was temp change before this param could be set from config files.  Will revert to original default.